### PR TITLE
Feature/wallet toggle and Home screen touch-ups [PIL-494]

### PR DESCRIPTION
**Pull request by vyvil**
This PR adds Wallet switcher to Home screen and addresses comments in PIL-494.

### CODE REVIEW

File: src/selectors/wallets.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    const { SMART_WALLET_ENABLED: smartWalletFeatureEnabled, BITCOIN_ENABLED: bitcoinFeatureEnabled } = featureFlags;
+    const keyWallet = accounts.find(({ type }) => type === ACCOUNT_TYPES.KEY_BASED) || {};
+    const availableWallets = [{ ...keyWallet, isActive: isBitcoinActive ? false : keyWallet.isActive }];
+
+    if (smartWalletFeatureEnabled) {
+      const smartWallet = accounts.find(({ type }) => type === ACCOUNT_TYPES.SMART_WALLET);
+      if (smartWallet) {
+        availableWallets.unshift({
+          ...smartWallet,
+          isActive: isBitcoinActive ? false : smartWallet.isActive,
```
Comment (grilix) [inline src/selectors/wallets.js:51]: I feel like without the ternary is easier to read, but not entirely sure (same for the key wallet). Your call:

```suggestion
          isActive: !isBitcoinActive && smartWallet.isActive,
```

File: src/screens/Home/Home.js [hunk 2]
```diff
@@ -452,7 +456,7 @@ class HomeScreen extends React.Component<Props, State> {
             <React.Fragment>
               <ScrollView
                 style={{ width: '100%', flex: 1 }}
-                stickyHeaderIndices={referralsFeatureEnabled ? [4] : [3]}
+                stickyHeaderIndices={referralsFeatureEnabled ? [3] : [2]}
```
Comment (grilix) [inline src/screens/Home/Home.js:459]: What do these numbers mean exactly? Can we put a name on them?

Comment (vyvil) [reply src/screens/Home/Home.js]: It means an index of items that should become sticky. This has been changed after ActivityFeed improvements

File: src/selectors/wallets.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
+// @flow
```
Comment (jsidorenko) [inline src/selectors/wallets.js:1]: We need to add the copyright/disclaimer here

**Pull request merged**